packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/updateModal.js

```diff
@@ -27,36 +28,46 @@ const UpdateModal = props => {
     return (
         <Modal>
             <aside className={rootClass}>
-                <Form className={classes.contents} onSubmit={handleSubmit}>
+                <Form
+                    className={classes.contents}
+                    getApi={setFormApi}
+                    initialValues={formInitialValues}
+                    onSubmit={handleSubmit}
+                >
                     <div className={classes.header}>
                         <span className={classes.headerText}>
                             {'Edit Shipping Method'}
                         </span>
-                        <UpdateModalCancelIconButton
-                            className={classes.headerButton}
-                            onClick={handleCancel}
-                        />
+                        {!isLoading && (
+                            <UpdateModalCancelIconButton
+                                className={classes.headerButton}
+                                onClick={handleCancel}
+                            />
+                        )}
                     </div>
                     <div className={classes.body}>
                         <ShippingRadios
                             isLoading={isLoading}
-                            selectedShippingMethod={selectedShippingMethod}
                             shippingMethods={shippingMethods}
                         />
                     </div>
                     <div className={classes.footer}>
-                        <UpdateModalCancelButton
-                            className={classes.footerCancelButton}
-                            onClick={handleCancel}
-                        />
-                        <Button
-                            className={classes.footerSubmitButton}
-                            priority="high"
-                            type="submit"
-                            disabled={pageIsUpdating}
-                        >
-                            {'Update'}
-                        </Button>
+                        {!isLoading && (
+                            <UpdateModalCancelButton
+                                className={classes.footerCancelButton}
+                                onClick={handleCancel}
+                            />
+                        )}
+                        {!isLoading && (
```

Comment:
And here

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2382/commits/ae1d682abbcd6dee3c0b7f071a3d32a7f78eafd1.